package Exception_Handling_day11;

public class TryCatch2 {

	public static void main(String[] args) {
		int data=5/0;
		System.out.println("no try catch ");

	}

}
